aspnetcore/security/authentication/social/google-logins.md

```diff
@@ -28,7 +99,7 @@ This tutorial shows you how to enable users to sign in with their Google account
 Store sensitive settings such as the Google `Client ID` and `Client Secret` with the [Secret Manager](xref:security/app-secrets). For the purposes of this tutorial, name the tokens `Authentication:Google:ClientId` and `Authentication:Google:ClientSecret`:
 
 ```dotnetcli
-dotnet user-secrets set "Authentication:Google:ClientId" "X.apps.googleusercontent.com"
+dotnet user-secrets set "Authentication:Google:ClientId" "<client id>"
```

Comment:
Why are you changing the old version?  `X.apps.googleusercontent.com` gives them a hint what it looks like.

Reply:
It was a bit confusing since we we do not do that for "Client Secret".  I ended up reading through it a few times trying to understand why that example treated ClientId differently to make sure I wasn't missing something and then realized it was purposely inconsistent.  I will change back if that logic does not seem sound.

Reply:
We generally don't update the previous version unless it's a glaring problem. Keep your approach if you feel strongly it's better.

Reply:
I will keep it.  I do get the problem with touching a previous version though, and very good point.  Thanks.